python/filtering/filter_menu_group.py

```diff
@@ -267,8 +274,36 @@ def add_to_menu(
             menu.addSeparator()
 
         if title:
-            title_action = menu.add_label(title)
-            self.header_action = title_action
+            # Add the filter group title menu entry (with actions)
+            reset_action = QtGui.QAction("Reset")
+            reset_action.triggered.connect(
+                lambda checked=False, m=menu: self.reset_filters(m)
+            )
+            remove_action = QtGui.QAction("Remove")
```

Comment:
I find the meaning of reset and remove quite confusing. Is it possible to clarify with more words (reset to default, remove all) and/or add tooltips with information?

Reply:
OK, that makes sense. Yes, please rename to "Reset Filter" and "Remove Filter". That will be enough for me.
Thanks.